UKHomeOffice/cerberus-service
# COP-10092: Update test payload with groupReference field

## Description
Update test payload with groupReference field

## To Test
npm run cypress:runner
run the following tests,
`Should create a task with a payload contains RoRo Tourist from RBT & SBT`
`Should create a task with a payload contains RoRo Unaccompanied Freight from SBT`

task should be created successfully.

## Developer Checklist

\* Required
